packages/sdk/ts/tally/commitment.ts

```diff
@@ -1,6 +1,6 @@
 import { genTreeCommitment, hash2, hash3, hashLeftRight } from "maci-crypto";
 
-import { IGenerateTallyCommitmentsArgs, ITallyCommitments } from "./utils/types";
+import type { IGenerateTallyCommitmentsArgs, ITallyCommitments } from "./types";
```

Comment:
maybe I'd call this file, commitments not commitment as there's more than one commitment for tally data